external/storm-hdfs/src/main/java/org/apache/storm/hdfs/bolt/AbstractHdfsBolt.java

```diff
@@ -52,9 +54,10 @@
      * Half of the default Config.TOPOLOGY_MESSAGE_TIMEOUT_SECS
      */
     private static final int DEFAULT_TICK_TUPLE_INTERVAL_SECS = 15;
+    private static final Integer DEFAULT_MAX_OPEN_FILES = 5;
```

Comment:
@rajkonda I like that idea, but there's any sort of fan out could happen between a spout and the HDFS Bolt.  I'll just start with a much larger number like 20.